websites/I/Indiachan/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence: Presence = new Presence({
+  clientId: "876813345699811328"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  }, p: string = document.location.pathname;
+  if (p === "/") {
+    presenceData.details = "Home";
+    presenceData.smallImageText = "Idling";
+    presenceData.smallImageKey = "dindu";
+    presence.setActivity(presenceData);
+    return;
```

Comment:
why set activity and return here

Reply:
to not make it go through the if else blocks below that

Reply:
well it's not like I don't know that but you check if the pathname is "/" so why does the code below matter other than the browsing one which is small image text anyways

Reply:
Because string functions like "includes" in the conditions are bound to make the code slower

Reply:
well, to my experience I wouldn't say so but sure if you say so, I won't argue

Reply:
Well I can just wrap the rest of the code in an else block if that's what you're asking me to do. But it won't be any different from using `return;` however.